locations/storefinders/storerocket.py

```diff
@@ -0,0 +1,35 @@
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours, sanitise_day
+from locations.items import Feature
+
+
+class StorerocketSpider(Spider):
+    id = ""
```

Comment:
Maybe this should be `storerocket_id` to make it clearer about how the ID is used?

Reply:
Yea, I think the primary use will be in other spiders that subclass from the `StorerocketSpider`, in which case it's helpful to have the extra info.